api/drivers/aws/driver.js

```diff
@@ -255,6 +295,67 @@ class AWSDriver extends Driver {
       });
     });
   }
+
+  /**
+   * Calculate credit used by a user
+   *
+   * @method getUserCredit
+   * @param {user} user The user how have close the session
```

Comment:
@param {Object}
"The user how have close the session" -> "User to calculate credit usage from"